### ISSUE TITLE
Issues with the MikTeX choco package

### ISSUE THREAD

**Issue by thistlillo** [MiKTeX/miktex-packaging#311]
- [x] I have read the [MiKTeX Contributing Guidelines](https://github.com/MiKTeX/miktex-packaging/blob/master/CONTRIBUTING.md)

I do not know if this is the correct place for this report or not.

On my new Windows 11 (Pro) PC, I had installed MikTeX via choco. Even if configured to install missing packages automatically, MikTeX was not doing it and I uninstalled it.

My PC was left in an inconsistent state: installing MikTeX via its normal installer (version 22.10-x64) resulted in several "inconsistencies between user/admin packages" and "internal errors" that prevented even the inspection of logs. After several attempts I was able to do a clean uninstalled followed by a correct installation

Now MikTeX is working correctly as it used to.

So, basically, MikTeX installed via choco might not work. In my case it didn't with errors very difficult to interpret.

**Comment by edocevoli** [comment]
Thank you for the information. I will keep this in mind.
